### ISSUE TITLE
Wide code-blocks are missing horizontal scroll bars

### ISSUE THREAD

**Issue by DenizThatMenace** [zulip/zulip-desktop#1045]
**Describe the bug**
Wide code-blocks in messages are missing horizontal scroll bars.  
Currently, if one wants to see the missing parts, one has to mark the text of the code-block and move the mouse to right.

**To Reproduce**
Send a message with a code-block that is wider than the width of the displayed message.

**Expected behavior**
I expect horizontal scroll bars for code-blocks that are wider than the message. Similar to how the Zulip web-app does it.

**Screenshots**
How it looks in the desktop app:
![Zulip-desktop-app_wide_code-block](https://user-images.githubusercontent.com/1121753/100211198-21e99000-2f0c-11eb-9730-b88921229d20.png)

How it looks in the web app (and how the desktop app should look like):
![Zulip-web-app_wide_code-block](https://user-images.githubusercontent.com/1121753/100211274-388fe700-2f0c-11eb-88c7-ff734f66e85d.png)

**Desktop (please complete the following information):**
 - Operating System: Kubuntu 20.04 x64
 - Zulip Desktop Version: 5.4.3

**Additional context**
Nope.

**Comment by andersk** [comment]
The scrollbars are there in both screenshots if you look closely. All you’re seeing is the Chrome/Electron styling vs. Firefox styling; Chrome supports scrollbar styling, while Firefox does not.

https://github.com/zulip/zulip/blob/5dc348916605ee56ab75ee289cd331d4581690bd/static/styles/rendered_markdown.css#L536-L556

**Comment by DenizThatMenace** [comment]
@andersk Thanks, you are right. However, I realized that I did not only get fooled by the colors but also by the mouse pointer.

In Firefox the mouse pointer is always the hand shown in the second image. Only if it hovers above the scrollbars it turns into an arrow. In the Zulip Desktop App and in Chrome the mouse pointer does not change into an arrow and always stays as the hand.

However, this is only the fact with these horizontal scrollbars within messages. Hovering above vertical scrollbars (e.g. for the message list) changes the mouse pointer into an arrow, even in Chrome or the Zulip Desktop App. (Then again, these vertical scroll bars look exactly as the scrollbars in Firefox shown in the second image above. The styles seem not to have effect on vertical scrollbars.)

I would suggest to harmonize the behavior and to always change the mouse pointer into an arrow when hovering above scrollbars.
